tests/test_message.py

```diff
@@ -30,31 +31,32 @@ async def test_message(self):
                 raw_message=raw_message,
             )
 
-            self.assertEqual(message.text, "Hello world")
-            self.assertEqual(message.user, "user")
-            self.assertEqual(message.target, "default")
-            self.assertEqual(message.raw_event["timestamp"], "01/01/2000 19:23:00")
-            self.assertEqual(message.raw_event["messageId"], "101")
+            assert message.text == "Hello world"
+            assert message.user == "user"
+            assert message.target == "default"
+            assert message.raw_event["timestamp"] == "01/01/2000 19:23:00"
+            assert message.raw_event["messageId"] == "101"
             with self.assertRaises(TypeError):
                 await message.respond("Goodbye world")
 
-    def test_depreacted_properties(self):
+@pytest.fixture
+def test_depreacted_properties(self):
         message = Message(text="hello", user="user", room="", connector="")
 
         message.target = "spam"
-        with self.assertWarns(DeprecationWarning):
+        with self.assertWarns == DeprecationWarning:
```

Comment:
I dont think we want this with statement here, you probably will need to look into the logging to see if the warning was emited

Reply:
where would I find this?

Reply:
where would I find that?